actions/deployments/deploy-to-test.yml

```
@@ -0,0 +1,59 @@
+name: Deploy to Test
+version: 0.0.0
+description: Deploy application to the test environment once changes have been pushed to main and built successfully.
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  DOCKER_IMAGE: ghcr.io/${{ github.repository }}:${{ github.run_number }}
+  KUBECONFIG: ${{ secrets.KUBECONFIG }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Download build artifact
+      run: docker build -t $DOCKER_IMAGE .
+      working-directory: .
+
+    - name: Save Docker image as artifact
+      run: docker save $DOCKER_IMAGE -o ${{ github.repository }}.tar
+
+    - name: Upload build artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: build-artifact
+        path: ${{ github.repository }}.tar
+        retention-days: 7
+
+  deploy-test:
+    # Needs build to be successful before deploying to test
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Download build artifact
+      uses: actions/download-artifact@v4
+      with:
+        name: build-artifact
+
+    - name: Load Docker image
+      run: docker load -i ${{ github.repository }}.tar
+      working-directory: .
+
+    - name: Deploy to Test Environment
```

Or just comment out line -57 "- name: Deploy to Test Environment"
```suggestion
    # - name: Deploy to Test Environment
```